src/controllers/requestController.js

```diff
@@ -0,0 +1,72 @@
+import model from '../database/models';
+import { findRooms } from '../services/roomService';
+
+// creating a request
+const createRequest = async (req, res) => {
+  try {
+    const { idUser, idRoom, dateStart, dateEnd } = req.body,
+      { id } = req.userData;
+    const existingRoom = await findRooms(idRoom);
+
+    if(!existingRoom) return res.status(400).json({ message: 'Room id does not exist' });
+    console.log(existingRoom);
+
+
+    const newRequest = {
+      idUser: req.userData.id,
+      idRoom,
+      dateStart,
+      dateEnd
+    };
+    const request = await model.request.create(newRequest);
+
+    console.log(idUser);
+  
+    return res.status(201).json({ message: 'Request created successfully!', request});
+  } catch (error) {
+    return res.status(500).json({error: error.message})
```

Comment:
Please avoid returning technical error messages to the user, they will not understand. a simple message like 'internal error, contact support, etc is good for the user. Technical errors can be logged using a preferred logging service for maintenance and support. Check all instances where you return error.message in the response.
Additionally, since these technical error messages are not known beforehand, they will undermine the multi-language support feature